### ISSUE TITLE
Hints not reachable from web view 

### ISSUE THREAD

**Issue by bmacho** [exercism/rust#1438]
Hey!

E.g. at this exercise https://exercism.org/tracks/rust/exercises/low-power-embedded-game/edit I can't see the available hints from the web view. 

When I click the stuck button I see
![image](https://user-images.githubusercontent.com/18089594/150945122-03ca2019-dc4b-4f98-bdb2-dd4e8168fc2a.png)
and when I click the light bulb at top right, I see
![image](https://user-images.githubusercontent.com/18089594/150945744-20d5f4e8-b1d4-4c5a-924e-ceae6263dc33.png)

Basically I only see the headlines of the hints.
There are additional hints written for this exercise: https://github.com/exercism/rust/blob/main/exercises/concept/low-power-embedded-game/.docs/hints.md

Is this broken for you too, or just me blocking some javascript something?

**Comment by senekor** [comment]
It's not broken for me. Is it still for you? If it is, this issue probably belongs in the [forum](https://forum.exercism.org/c/support/8). It's likely not related to the Rust track specifically.
